staging/src/k8s.io/cri-client/pkg/logs/logs_test.go

```diff
@@ -363,12 +364,12 @@ func TestParseLog(t *testing.T) {
 		t.Logf("TestCase #%d: %+v", c, test)
```

Comment:
perhaps wrapping this in `t.Run()` will eliminate the warning and will make those unit test run in parallel.